tests/test_view_data_cli.py

```diff
@@ -189,7 +189,7 @@ def test_do_run_sql(self, patched_prompt):
             in output
         )
         assert (
-            "| NMEA_bad.log                     | None |  243 | 8ddb840fee218872d2bb394cc654bdae"
+            "| NMEA_bad.log                     | None |  243 | b201a229fb2c4a80bd657066e4bf9c8a"
             in output
```

Comment:
@robintw  - has the contents of `NMEA_bad.log` changed?  Do you understand why the hash has changed?

Reply:
It changed in one of the other PRs, so when I pulled from `develop` I got the new version of the `NMEA_bad.log` file, and had to update the hash.

That file changed because in one of the other PRs adding tests I added a blank line to the file to test that the parser ignored blank lines.

Reply:
As expected, thanks.